static/tests/users.csv

```diff
@@ -1,10 +1,8 @@
 "name","pubkey","joinedAt","tx","type","createdAt","updatedAt"
 "0xd44a82dD160217d46D754a03C8f841edF06EBE3c","dBgXJATrP4KeE6zfuR4_arauMIeT_86MrQg6JbbnuxM.yJXykCW6qjB54B29by8vIWoMwk8T5NG_3awHdKC9Bgc",1644251733000,"0x9e532171096cf4f4fe68cd384addf3baaf31644c48506cb4550efb586a165c5a","arbitrum","2022-03-17 12:24:25.866+08","2022-03-17 12:24:25.866+08"
 "0x3E1E26f055Cd29053D44Fc65aa1FCa216DedFceb","MNw7njaTh0k835aq0JKtmpq33izkGwFxdldqf3txB64.a-yzwTFi1hNP-4lrpHB5NAw7p100oAOUefpYwfLPer8",1644254074000,"0xfde1d05921ccb073f71b747fd323fa313d51c77db82311993555692881ff9387","arbitrum","2022-03-17 12:24:26.103+08","2022-03-17 12:24:26.103+08"
-"0x860C68abeEE0ecBd97729C7d1E09dA1e7EaD3C50","",,"","","2022-03-17 12:24:35.267+08","2022-03-17 12:24:35.267+08"
```

Comment:
table inserts would fail because schema doesn't allow for null values
https://github.com/zkitter/zkitterd/blob/30824fd6afdc8ed4503fbba1117f21fde3852c77/src/models/users.ts#L51